src/dirtyfields/dirtyfields.py

```diff
@@ -17,6 +17,10 @@ class DirtyFieldsMixin(object):
     # https://github.com/romgar/django-dirtyfields/issues/73
     ENABLE_M2M_CHECK = False
 
+    # Flag to stop resetting the state on every M2M change, only reset after
+    # post_save(), saving us DB queries
+    RESET_STATE_ON_SAVE_ONLY = False
```

Comment:
I know you might've renamed this already, but I think `ENABLE_BASIC_M2M_CHECK` might be a better name?

`RESET_STATE_ON_SAVE_ONLY` doesn't communicate that it's related to m2m, not what's really going on.